doc/schema/libdoc_schema.json

```diff
@@ -247,38 +257,50 @@
       "title": "TypedDict Type schema",
       "required": [
         "name",
-        "super",
+        "type",
         "doc",
-        "items",
-        "required_keys",
-        "optional_keys"
+        "items"
       ],
       "properties": {
         "name": {
           "type": "string"
         },
-        "super": {
+        "type": {
           "type": "string"
         },
         "doc": {
           "type": "string"
         },
         "items": {
-          "type": "object"
-        },
-        "required_keys": {
           "type": "array",
           "items": {
-            "type": "string"
+            "$ref": "#/definitions/typedDictItem"
           },
           "uniqueItems": true
+        }
+      }
+    },
+    "typedDictItem": {
+      "$id": "#/definitions/typedDictItem",
+      "type": "object",
+      "title": "TypedDict Item schema",
+      "required": [
+        "key",
+        "type",
+        "required"
+      ],
+      "properties": {
+        "key": {
+          "type": "string"
         },
-        "optional_keys": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
-          "uniqueItems": true
+        "type": {
+          "type": "string"
+        },
+        "required": {
+          "type": [
+            "boolean",
+            "null"
```

Comment:
I would rather omit `required` altogether than set it `null`, but I guess using `null` has been the approach used with JSON specs in general.

Reply:
yes it is omitted in XML and we do not omit anything in json